docs/nodes/nodes-library/core-nodes/SSH/README.md

```diff
@@ -0,0 +1,68 @@
+---
+permalink: /nodes/n8n-nodes-base.ssh
+description: Learn how to use the SSH node in n8n
+---
+
+# SSH
+
+The SSH node is useful for executing commands via the Secure Shell Protocol.
+
+::: tip 🔑 Credential
+Authentication for the SSH node requires that you have a username and password for connecting to the remote server, or an SSH key configured for the server or service you are connecting to. See [Connecting to GitHub with SSH] for an example](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh).
```

Comment:
This doesn't render correctly. Can we fix the markdown issue for the Link?

Reply:
Can we also add separate documentation for the credentials? Example: https://docs.n8n.io/credentials/smtp/#send-email

Reply:
Done 👍  Though in general I disagree with the practice of adding a credential page regardless of the node - the Send Email one being a perfect example. That page adds no value - the same could be accomplished in one line in the node docs, and without forcing the user to navigate away to a separate page.

Reply:
This helps people who are not technical. If you check the IMAP node, we added docs for different services. In the future, if the community is facing issues with creating credentials and connecting the SSH node, this is where the documentation would go 🙂